src/proxy-ui-api/frontend/tests/e2e/specs/ss-backupandrestore.js

```diff
@@ -135,7 +135,7 @@ module.exports = {
     backupAndRestoreTab.enterFilterInput(createdBackupFileName);
     browser.expect
       // xrd-table has thead but no tbody so use '/tr' to get only the row elements that are direct children of table
-      .elements('//table[contains(@class, "xrd-table")]/tr')
+      .elements('//table[contains(@class, "xrd-table")]/tbody/tr')
```

Comment:
This code no longer matches the comment above

Reply:
Removed the comment